<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!-- Config file -->
  <arg name="debug_file"             default="$(find graph_monitor)/config/debug.yaml"/>

  <node name="msg_converter" pkg="graph_monitor" type="msg_converter.py" output="screen" respawn="true">
    <rosparam command="load" file="$(arg debug_file)"/>
  </node>
</launch>
